From 8f5d28536e4518716fdfe974e580194c8f57871d Mon Sep 17 00:00:00 2001
From: Chris Liddell <chris.liddell@artifex.com>
Date: Thu, 7 Jan 2016 09:03:10 +0000
Subject: [PATCH] Bug 696281: fix check for using shared freetype lib

When I changed the initial value of the Freetype source path variable (to reduce
the risk of header search path problems), I neglected to fix the logic for
falling back to the system's libfreetype2.

Credit to Rodrigo Rivas Costa for spotting the problem.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 0c97fcc..f533e46 100644
--- a/configure.ac
+++ b/configure.ac
@@ -811,7 +811,7 @@ if test x"$enable_fapi" != xno; then
       fi
     done
 
-    if test -z $FTSRCDIR; then
+    if test x"$FTSRCDIR" = x"src"; then
       AC_MSG_RESULT([no])
       if test "x$PKGCONFIG" != x; then
         AC_MSG_CHECKING(for system freetype2 >= 2.4.2 with pkg-config)
-- 
2.6.3

